packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -170,15 +169,15 @@ class IconButton extends StatelessWidget {
 
   /// The size of the icon inside the button.
   ///
-  /// This property must not be null. It defaults to 24.0.
+  /// If [iconSize] is null, then it defaults to 24.0.
```

Comment:
This is a misleading statement. If `iconSize` is null, but `IconTheme.of(context).size` is provided, it may not be 24.0.